Apps/VisionOS/VisionOS/APIViews/TrackEventsView.swift

```diff
@@ -0,0 +1,74 @@
+import MarkdownUI
+import SwiftUI
+
+private func eventAsCodeSnippet(_ event: Event) -> String {
+    let propertiesStr = propertiesToTutorialString(
+        event.properties,
+        linePrefix: "\t\t"
+    )
+
+    if !event.name.isEmpty, !propertiesStr.isEmpty {
+        return
+            """
+            CustomerIO.shared.track(
+                name: "\(event.name)",
+                properties: \(propertiesStr)
+            )
+            """
+    } else if !event.name.isEmpty {
+        return
+            """
+            CustomerIO.shared.track(
+                name: "\(event.name)"
+            )
+            """
+    }
+    return "// enter event name"
+}
+
+struct TrackEventsView: View {
+    @EnvironmentObject private var viewModel: ViewModel
+
+    @State private var event = Event(name: "", properties: [])
+
+    let onSuccess: (_ event: Event) -> Void
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Markdown {
+                """
+                Using CustomerIO you can track events for the identified person using the `track` API.
```

Comment:
With 3.x  we [can track events for anonymous persons](https://customer.io/docs/sdk/ios/tracking/track-events/#anonymous-activity) too i.e. if an event is tracked before identifying a person, it will be assigned an `anonymousId`